Doberman/SensorMonitor.py

```diff
@@ -99,4 +99,4 @@ def reload_readings(self):
             self.readings[reading_name] = Doberman.Reading(self.name, reading_name,
                                                            self.db)
             self.register(func=self.ScheduleReading, period=r.readout_interval,
```

Comment:
Unrelated, but I don't think this line works. What is `r`?

Reply:
I thought I had already rewritten that function but that was probably lost in the Great Push